### PR TITLE
mac os x : main menubar with exit menu for darwin build

### PR DESCRIPTION
**Pull request by pravic**
@mchampaneri I've merged your PR from #232 into a separate branch, because #227, #228, #229, #230, #231 together would be a mess. This one is quite clear and will go directly to master.

refs #227 
closes #232

### CODE REVIEW

File: examples/demoes/09/main.go [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"github.com/sciter-sdk/go-sciter"
+	"github.com/sciter-sdk/go-sciter/window"
+)
+
+func main() {
+	win, _ := window.New(sciter.DefaultWindowCreateFlag, sciter.DefaultRect)
+	win.AddQuitMenu()
```
Comment (pravic) [inline examples/demoes/09/main.go:10]: Perhaps, it will be better to add some explanation here.

Comment (mchampaneri) [reply examples/demoes/09/main.go:10]: Yes, 
I have added details about AddQuitMenu() and window. Code updated  with required explanation.

File: examples/demoes/09/main.go [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"github.com/sciter-sdk/go-sciter"
+	"github.com/sciter-sdk/go-sciter/window"
+)
+
+func main() {
+	win, _ := window.New(sciter.DefaultWindowCreateFlag, sciter.DefaultRect)
```
Comment (mchampaneri) [inline examples/demoes/09/main.go:9]: // Creating application window with default flags and rect size

File: examples/demoes/09/main.go (hunk 1 above)
Comment (mchampaneri) [reply examples/demoes/09/main.go:10]: // AddQuitMenu function adds Quit menu (in menubar) and support for
	// CMD+q to terminate / close application on MacOS
	//
	// Note : AddQuitMenu() only works for MacOS on other platform is
	//        does not have any effect.

**Pull request merged**